ChromecastSampleApp/ChromecastSampleApp/Players/PlayerViewController.m

```diff
@@ -98,6 +98,7 @@ - (void) notificationHandler:(NSNotification*) notification {
   }
   if ([notification.name isEqualToString:OOOoyalaPlayerPlayCompletedNotification] && self.embedCode2) {
     [self play:self.embedCode2];
+    self.embedCode2 = nil;
```

Comment:
It is the trick used to create the "2 Assets autoplayed" test. All other assets only have embedCode, and embedCode2 is nil for them. When playback finishes, we want to play embedCode2 automatically, but then not play anything else afterwards. W/out this change, the 2nd asset was playing and replaying for ever.